### ISSUE TITLE
Modeling variant effect on response to 'combination' treatments

### ISSUE THREAD

**Issue by mbrush** [NCATS-Tangerine/ncats-ingest#28]
Hi @goodb @stuppie @putmantime.  Wanted to reach out as we are gearing up to ingest several cancer variant data sources, as per ticket #27.  Would be nice to coordinate efforts as much as possible - as we work through the tasks of data exploration, parsing, modeling, and identifier mapping/resolution.
 
I had an initial question for WD folks as I started poking around Civic and drafting some potential models for representing variant effect on susceptibility to treatment.  Specifically, in cases where the treatment is a combination therapy involving more than one drug. Here, it looks like WD creates separate claims between the variant and each drug in a combination - is this accurate?  For example, consider the Civic statement 'EID98' [here](https://civic.genome.wustl.edu/events/genes/5/summary/variants/12/summary/evidence/816/summary#evidence), claiming that the BRAF V600E variant correlates with sensitivity of colorectal cancer to combination treatment with Capecitabine, Vemurafenib, and Bevacizumab: 
![civic-eid98](https://cloud.githubusercontent.com/assets/5184212/26327953/10ac7e7e-3ef6-11e7-8461-60de59ba7549.jpg)

From this, WD creates **three separate claims** about the V600E variant [here](https://www.wikidata.org/wiki/Q21851559) - stating that it is a positive therapeutic predictor for colorectal cancer response to Capecitabine, Vemurafenib, and Bevacizumab **individually**. The claim for Capecitabine is shown below:
![civic-eid_098-wd](https://cloud.githubusercontent.com/assets/5184212/26327973/236b1142-3ef6-11e7-8344-b61b570ec0c1.jpg)

This approach of making separate assertions for each drug in a combination would obviously simplify the modeling - allowing for direct links form a variant item to a drug item as above, instead of having to model the concept of a combination therapy. **But it may be problematic to imply that because a variant conferred sensitivity to treatment with drug A + B + C, we can state that it confers sensitivity to drug A on its own**.  

Do the WD folks have a sense of if/how deeply this was considered when Civic was ingested? Was there some agreement among experts that it was acceptable/valid to make this simplification from the clinical perspective? Or was it a matter of prioritizing as simple a representation as simple as possible to fit the WD model, even if some nuance is lost?
 
Thanks . . . and please pass this along to any other relevant WD/BT folks who may have insight.  
Mentioning @pnrobinson and @mellybelly as they may have thoughts here as well.

**Comment by stuppie** [comment]
Adding @andrawaag to the conversation

**Comment by andrawaag** [comment]
Hi @mbrush @goodb @stuppie @putmantime. The WD items should reflect the CIViC record, so I agree the combination should not be modelled as separate assertions. The bot code needs some rewiring to fix this and I am working on it.

**Comment by stuppie** [comment]
@mbrush Looking at the API response from CIViC, it doesn't look like they distinguish between combination drugs and a list of drugs (or, not and). For example, the api response for the example above ([link](https://civic.genome.wustl.edu/api/evidence_items/98)), which is referring to the combination of Vemurafenib, Capecitabine, and Bevacizumab, has the same structure as another evidence item which is specifically dacarbazine or temozolomide ([link](https://civic.genome.wustl.edu/api/evidence_items/82)).

**Comment by mbrush** [comment]
@stuppie There is an attribute called _drug_interaction_type_ that is used in cases where there is more than one agent in the drugs list, to indicate if the treatment  in question refers to any of the drugs individually, or all the drugs in combination.  I have noticed in looking at records in their web app that this attribute is sometimes empty - which happens to be the case in the two examples you shared above.  

[Here ](https://civic.genome.wustl.edu/api/evidence_items/1411) is a record for another statement that refers to a combination treatment with many drugs at once, and the value of the _drug_interaction_type_ attribute is '**Combination**'.  [Here ](https://civic.genome.wustl.edu/api/evidence_items/816) is a record for another statement  that refers to separate treatments with any of the drugs on the list individually, and the value of the _drug_interaction_type_ attribute is '**Substitutes**'.

So their model should support the distinction - but unfortunately the relevant attribute seems in some cases to be null.  It would be nice to know how frequently this is the case (i.e. in what % of records where there is more than one drug is there no value for this attribute). I'd guess you could figure out pretty quickly?  Then we may need to write a ticket for CIViC to go back and re-curate these records.  From what I've seen, it is always clear from the free-text description whether the indicated drugs are applied in combination or separately - so should be easy for them to fix records where a structured annotation to this effect was not made.

**Comment by kkrysiak** [comment]
After looking at this discussion, we are making the _drug_interaction_type_ field a required field in CIViC (https://github.com/genome/civic-client/issues/763). I'm going to take a crack at filling in all of the missing ones. I also want to ping @acoffman @jmcmichael here so they are aware of this effort.

**Comment by mbrush** [comment]
Thanks @kkrysiak! @stuppie @andrawaag  - will WD update its model to incorporate the notion of combination therapies?  Do you have a sense of how this would look? We are exploring modeling approaches for Dipper/SciGraph but havent settled on anything yet.

**Comment by goodb** [comment]
Just for clarity here, @mbrush , when you ask "will WD update its model" that really means (in this case) "will @andrawaag @stuppie and team update the model?".  Its really up to us to lead here.  So, if you have ideas about how to express this knowledge given the wikidata structure, would love to hear them.  I suspect that at some point it will be unavoidable and not that bad to produce distinct combination therapy entities.  Something like (E used to treat Diseas1, E instanceOf ComboTherapy , E hasPart drug1,2,3).

**Comment by stuppie** [comment]
@mbrush Yes, we will update the model. We'll have to discuss the best way to model it... 
I think the best way would be to create "combination therapy" items, where this specific instance would be "Capecitabine, Vemurafenib, and Bevacizumab Combination Therapy". This item would "has part" the three drugs. 
As a comparison, there are many drug mixtures on Wikidata, where a drug mixture is a pharmaceutical product that contains two or more active ingredients. Here is an example: https://www.wikidata.org/wiki/Q7697766 .

**Comment by mbrush** [comment]
@goodb - yes, exactly.  Apologies I sometimes forget that WD is a much larger endeavor than the biomedical data you guys are working on.  The idea of creating combination therapy entities is one approach we are considering for SciGraph.  A Dipper ticket about this is [here](https://github.com/monarch-initiative/dipper/issues/463)  - note some of the nuances w.r.t. things like punning that distinguish proposals 2A and 2B are perhaps less relevant for the WD model. 

Structurally, proposal 2 (**model 2B** in particular) is similar to what @goodb and @stuppie outline above.  But we are considering modeling the combination therapies as _processes (exposures)_ rather than collections of material entities, to be consistent with and leverage the [ECTO ontology](https://github.com/cmungall/environmental-conditions) being developed by @cmungall.  Here, a combination therapy would have each participating drug as an _input_  rather than as a _part_.  

Of course an approach that models combination therapies as material collections with each drug as a part is equally valid - and would be more consistent with current representations of pharmaceutical mixtures as you note above.  So, we are not committed to anything here yet - just exploring some options, and awaiting feedback from Chris and others about an exposure-based, ECTO-aligned based approach.
